estate/views/estate_property_tag_views.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="estate_properties_tag_form" model="ir.ui.view">
+        <field name="name">estate.property.tag.form</field>
+        <field name="model">estate.property.tag</field>
+        <field name="arch" type="xml">
+            <form string="Tag">
+                <h1>
+                    <field name='name' />
+                </h1>
```

Comment:
That way, the label doesn't show up. Maybe better enclose it in a `<div oe_title>`

Reply:
put a placeholder instead, but it looks cool